watch-package.js

```diff
@@ -120,6 +124,8 @@ function startScript(script, pkg, processes) {
     var args = extensions ? ['--ext', extensions] : []
     args = args.concat(patterns)
     if (ignores) { args = args.concat(ignores) }
+    if (legacyWatch) { args = args.concat(['--legacy-watch']) }
+    if (delay) { args = args.concat(['--delay', delay]) }
```

Comment:
With the concatenation done above this will always have a value, may I suggest either expanding the check here, or only doing the initial concat if a value was set?